fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -3,14 +3,17 @@
       host "#{ENV['ES_COPY_HOST']}"
       port "#{ENV['ES_COPY_PORT']}"
       scheme "#{ENV['ES_COPY_SCHEME']}"
-      index_name ${record['kubernetes_namespace_name']}.${record['kubernetes_namespace_id']}.${Time.at(time).getutc.strftime(@logstash_dateformat)}
+      index_name project.${record['kubernetes']['namespace_name']}.${record['kubernetes']['namespace_id']}.${Time.at(time).getutc.strftime(@logstash_dateformat)}
       user "#{ENV['ES_COPY_USERNAME']}"
       password "#{ENV['ES_COPY_PASSWORD']}"
 
       client_key "#{ENV['ES_COPY_CLIENT_KEY']}"
       client_cert "#{ENV['ES_COPY_CLIENT_CERT']}"
       ca_file "#{ENV['ES_COPY_CA']}"
 
+      type_name ${(record && record.delete('es_type')) || 'fluentd'}
```

Comment:
Actually `type_name` is set where `es_type` is set here https://github.com/openshift/origin-aggregated-logging/pull/234/files#diff-5871732c8b4b021e8bebf8e5352d6deaR29